UNRELEASED_CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 - Add new attrs to `MessageListView`: `streamDeleteMessageActionEnabled`, `streamEditMessageActionEnabled`
 - Improve Channel List Diff
 - Add new attrs to `MessageInputView`: `streamInputScrollbarEnabled`, `streamInputScrollbarFadingEnabled`
+- Add API for setting custom message date formatter in MessageListView via `setMessageDateFormatter(DateFormatter)`
+    - 24 vs 12 hr controlled by user's System settings.
```

Comment:
For custom formatters, this system settings part doesn't apply, they choose how to format it from just a `LocalDateTime`